prawns/governance/src/handlers/typeGetters/getBountiesBountyProposedEvent.ts

```diff
@@ -0,0 +1,48 @@
+import { SubstrateNetwork } from '../../model';
+import { BountiesBountyProposedEvent as KusamaBountiesProposeEvent } from '../../types/kusama/events';
+import { BountiesBountyProposedEvent as PolkadotBountiesProposeEvent } from '../../types/polkadot/events';
+import { BountiesBountyProposedEvent as KhalaBountiesProposeEvent } from '../../types/polkadot/events';
+import { EventHandlerContext } from "@subsquid/substrate-processor/lib";
+
+export function getBountiesBountyProposedEvent(
+  ctx: EventHandlerContext,
+  network: SubstrateNetwork
+): {
+  index: number;
+} {
+  switch (network) {
+    case SubstrateNetwork.kusama: {
+      const event = new KusamaBountiesProposeEvent(ctx);
+
+      if (event.isV2028) {
+        return { index: event.asV2028 };
+      } else {
+        return event.asLatest;
+      }
+    }
+
+    case SubstrateNetwork.polkadot: {
+      const event = new PolkadotBountiesProposeEvent(ctx);
+
+      if (event.isV28) {
+        return { index: event.asV28 };
+      } else {
+        return event.asLatest;
+      }
+    }
+
+    case SubstrateNetwork.phala: {
+      const event = new KhalaBountiesProposeEvent(ctx);
+
+      if (event.isV28) {
+        return { index: event.asV28 };
+      } else {
+        return event.asLatest;
+      }
+    }
```

Comment:
I think you should include `V9130` check here as well. 
Even though now it's the same as the latest, when a new latest comes in, it might not be the latest anymore 😂